spec/controllers/generic_works_controller_spec.rb

```diff
@@ -136,12 +134,8 @@
                                  file_set: {}
           }.to change(FileSet, :count).by(2)
           created_files = FileSet.all
-          ["https://dl.dropbox.com/fake/blah-blah.Getting%20Started.pdf", "https://dl.dropbox.com/fake/blah-blah.filepicker-demo.txt.txt"].each do |url|
-            expect(created_files.map(&:import_url)).to include(url)
-          end
-          ["filepicker-demo.txt.txt", "Getting+Started.pdf"].each do |filename|
-            expect(created_files.map(&:label)).to include(filename)
-          end
+          expect(created_files.map(&:import_url)).to include(url1, url2)
```

Comment:
I agree that loops should be removed. I don't think that a DRY test is always the easiest to understand.